src/components/ResultsPage/ResultsBody/ResultsBody.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import './ResultsBody.css'
+
+function ResultsBody() {
```

Comment:
nit: we should use arrow functions instead of this syntax for the sake of consistency